src/openrct2/windows/ride_list.c

```diff
@@ -945,3 +953,100 @@ static void window_ride_list_open_all(rct_window *w)
         ride_set_status(i, RIDE_STATUS_OPEN);
     }
 }
+
+#pragma region Quick demolish prompt
```

Comment:
Wouldn't this be better off in it's own file? Easier to manage that way I'd say. Happy to be told otherwise however!